src/components/employees.js

```diff
@@ -4,50 +4,58 @@ import EmployeeCard from "./employeeCard";
 function Employees() {
   const [employeesData, setEmployeesData] = useState([]);
 
-  var Airtable = require('airtable');
-  var base = new Airtable({
-    apiKey: 'pat9Ta5mPLqZiE3gt.57a08925d5f2630489cc04cec28fe22a38b7593f53de573e2675fd437151aa7d'})
-    .base('appvtPVdJIF7ODnQF');
+  const Airtable = require("airtable");
+  const base = new Airtable({
+    apiKey:
+      "pat9Ta5mPLqZiE3gt.57a08925d5f2630489cc04cec28fe22a38b7593f53de573e2675fd437151aa7d",
+  }).base("appvtPVdJIF7ODnQF");
 
-  base('Employee directory')
-    .select({
-      fields: ["Full Name", "Job Title", "Department"],
-      view: "All employees",
-    })
-    .eachPage(function page(records, fetchNextPage) {
-      // This function (`page`) will get called for each page of records.
-      const data = records.map(function(record) {
-          console.log('Retrieved', record.get('Full Name'));
-          let currentEmployee = {
-            name: record.get("Full Name"),
-            title: record.get("Job Title"),
-            department: record.get("Department"),
-          };
-          return currentEmployee;
-      });
-      setEmployeesData(data); //this causes an undefined error
+  useEffect(() => {
```

Comment:
I had to add the `useEffect` back...a very fun infinite loop happens without it since the `useState` infinitely gets called and the component re-renders 😄